CODINGSTYLE.md

```diff
@@ -3,9 +3,10 @@
 The Scala.js project has a strict policy regarding coding style.
 This is one of the cornerstones that has allowed Scala.js to maintain a clean, consistent and maintainable codebase.
 
-This document tries to document the style in use as much as possible to make it easier for everyone to contribute.
+Most coding style rules can be applied automatically by scalafmt. It is recommended that you configure your editor to reformat on save; see the [scalafmt instructions](https://scalameta.org/scalafmt/docs/installation.html) for how to configure this.
```

Comment:
Consider putting all this in the "Add scalafmt" commit.

Reply:
Done